extensions/crc/scripts/build.js

```diff
@@ -19,13 +19,13 @@
 
 const zipper = require('zip-local');
 const path = require('path');
-const package = require('../package.json');
+const packageJson = require('../package.json');
```

Comment:
I don't see why these changes are related to code coverage ?

Reply:
it can be a separate PR if you prefer to rename some constants but it should not be related to code coverage

Reply:
At the begging, some of the coverage engines complained about keyword, now it is gone with c8 coverage engine, so reverting that.